### DISCUSSION TITLE
Subtitles font won't change

### DISCUSSION THREAD

**Discussion by Skaell** [iina/iina#3759] [Q&A]
Hi, 
I'm currently experiencing a little problem where when I try to change my subtitles font, it just stays the same as the base one. I tried relaunching IINA, but nothing seems to work. What should I do?

**Comment by low-batt** [comment]
IINA 1.3.0 has been released. It can be downloaded from the official [IINA](https://iina.io/) website.
The release includes updates to mpv and other libraries involved with subtitles.
I just tried changing the subtitles font and it worked for me.
Give 1.3.0 a try. If you still experience a problem enter an issue with details on how to reproduce and log files.
